src/css/utilities.css

```diff
@@ -0,0 +1,288 @@
+@utility backface-hidden {
+  backface-visibility: hidden;
+}
+
+@utility break-words {
+  word-break: break-word;
+}
+
+@utility text-vertical-lr {
+  writing-mode: vertical-lr;
+}
+
+/**
+  Link color utilities
+*/
+@utility link-* {
+  a {
+    color: --value(--color-*);
+  }
+}
+
+@utility link-hocus-* {
+  a:hover,
+  a:focus {
+    color: --value(--color-*);
+  }
+}
+
+/**
+  Link underline utilities
+*/
+@utility link-underline {
+  a {
+    text-decoration: underline;
+  }
+}
+
+@utility link-no-underline {
+  a {
+    text-decoration: none;
+  }
+}
+
+/**
+  Link font weight utilities
+*/
+@utility link-normal {
+  a {
+    font-weight: var(--font-weight-normal);
+  }
+}
+
+@utility link-semibold {
+  a {
+    font-weight: var(--font-weight-semibold);
+  }
+}
+
+@utility link-bold {
+  a {
+    font-weight: var(--font-weight-bold);
+  }
+}
+
+/**
+  Responsive spacing - padding
+*/
+@utility rs-p-* {
```

Comment:
Interesting. So this is how they do their dynamic property values. Simple enough.

Reply:
Yeah I'm glad they have these - compensate for no js 👍🏼